python/nnvm/frontend/darknet_c_interface.py

```diff
@@ -0,0 +1,412 @@
+"""
+Compile DarkNet Models
+====================
+**Author**: `Siju Samuel <https://github.com/siju-samuel/>`_
+
+DarkNet c interface frontend.
+
+"""
+from enum import IntEnum
+from cffi import FFI
```

Comment:
this seems only needed for evaluation purposes, avoid this in the front end package itselt

Reply:
We need to make sure user can use this with minimum dependency